Gemfile

```diff
@@ -40,3 +40,9 @@ group :development, :test do
   gem "rubocop-rails-omakase", require: false
   gem "rspec-rails", "~> 6.1.0"
 end
+
+gem "rest-client", "~> 2.1"
+
+gem "dotenv", "~> 3.1"
+
+gem "pry", "~> 0.14.2"
```

Comment:
Do we need pry? We have a debug gem. Also pry shouldn't be in main group, but in dev/test

Reply:
I will move

Reply:
removed